kernel/arch/dreamcast/include/dc/g2bus.h

```diff
@@ -31,50 +32,184 @@
 #include <sys/cdefs.h>
 __BEGIN_DECLS
 
```

Comment:
GitHub won't let me tag the exact line, but don't forget to add `\author Andy Barajas` to line 27.

Reply:
Done